src/vue/pages/Issuance.vue

```diff
@@ -0,0 +1,204 @@
+            <tr v-for="issuance in issuanceHistory" :key="issuance.id">
+              <td v-if="issuance.counterparty">
+                {{ issuance.counterparty }}
+              </td>
+              <td v-else>
+                <loader
+                  :message-id="'issuance.lbl-loading'"
+                />
+              </td>
+              <td>
+                {{
+                  issuance.amount | formatMoney
+                }}
+              </td>
+              <td>
+                {{
+                  issuance.asset
+                }}
+              </td>
+              <td>
+                {{ issuance.ledgerCloseTime | formatCalendar }}
+              </td>
+              <td>
+                {{ issuance.reference }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div v-else-if="!isLoadingFailed">
+      <loader
+        :message-id="'issuance.lbl-loading'"
+      />
+    </div>
+    <div v-else>
+      <p>
+        {{ 'issuance.lbl-loading-error' | globalize }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+import Loader from '@/vue/common/Loader'
+import Drawer from '@/vue/common/Drawer'
+import TopBar from '@/vue/common/TopBar'
+import CreateIssuanceForm from '@/vue/forms/CreateIssuanceForm'
+import UploadPreIssuanceForm from '@/vue/forms/UploadPreIssuanceForm'
+
+import { Sdk } from '@/sdk'
+// FIXME: move XDR-dependent object imports to sdk
+import { ACCOUNT_TYPES, OPERATION_TYPES } from '@/js/const/xdr.const'
+
+import { mapGetters } from 'vuex'
+import { vuexTypes } from '@/vuex'
+
+import { errors } from '@tokend/js-sdk'
+
+const PAGE_LIMIT = 200
+
+export default {
+  name: 'issuance',
+  components: {
+    Loader,
+    Drawer,
+    TopBar,
+    CreateIssuanceForm,
+    UploadPreIssuanceForm
+  },
+  data: _ => ({
+    issuanceHistory: null,
+    isLoaded: false,
+    isLoadingFailed: false,
+    isCreateDrawerShown: false,
+    isPreIssuanceDrawerShown: false,
+    ACCOUNT_TYPES
+  }),
+  computed: {
+    ...mapGetters([
+      vuexTypes.account
+    ])
+  },
+  async created () {
+    await this.loadIssuanceHistory()
+    this.isLoaded = true
+    await this.loadCounterpartyEmails()
+  },
+  methods: {
+    async loadIssuanceHistory () {
+      try {
+        // FIXME: Add pagination
+        const response = await Sdk.horizon.operations.getPage({
+          account_id: this[vuexTypes.account].id,
+          operation_type: OPERATION_TYPES.createIssuanceRequest,
+          limit: PAGE_LIMIT
+        })
+        this.issuanceHistory = response.data
+      } catch (error) {
+        this.isLoadingFailed = true
+        console.error(error)
+      }
+    },
+    getCounterpartyId (issuance) {
+      const participants = issuance.participants
+      return participants[0].accountId === this[vuexTypes.account].id
+        ? participants[1].accountId
+        : participants[0].accountId
+    },
+    async loadCounterpartyEmails () {
```

Comment:
Let's write the `email-getter` component that accepts the account or balance ID, then loads email and renders it to the inner `span` element

Reply:
so we can avoid this awful reactivity hacks

Reply:
we already have something similar in admin-panel